packages/manager/apps/pci-block-storage/src/api/data/instance.ts

```diff
@@ -36,3 +36,7 @@ export const getInstance = async (
   );
   return data;
 };
+
+export function canAttachVolume(instance: Instance): boolean {
```

Comment:
I would have moved this function to the hook file as it's not related to any I/O.

BTW, this value could be returned directly by the hook `useInstances` itself, as a getter.